src/components/Node.js

```diff
@@ -118,7 +118,7 @@ const Node = ({ type, name, path, parentCommitmessage, level }) => {
           style={{
             paddingLeft: INDENT_SIZE * level + 10,
             display: 'grid',
-            gridTemplateColumns: '18px 18px 1fr',
+            gridTemplateColumns: '26px 26px 1fr',
```

Comment:
Feel free to play with these numbers to match the design refresh; I didn't spend much time on this.